pkg/console/controllers/clidownloads/controller.go

```diff
@@ -186,6 +186,11 @@ func PlatformBasedOCConsoleCLIDownloads(host, cliDownloadsName string) *v1.Conso
 		})
 	}
 
+	links = append(links, v1.CLIDownloadLink{
+		Href: fmt.Sprintf("%s/oc-license", baseURL),
+		Text: fmt.Sprintf("Download LICENSE for OpenShift Command Line Interface"),
```

Comment:
Maybe just "LICENSE"? Shouldn't need `Sprintf`

```suggestion
		Text: "LICENSE",
```